scala/build/Dockerfile.latest

```
@@ -0,0 +1,102 @@
+chown -R 1000:1000 /home/qa
+
+# Set work directory
+WORKDIR /home/qa/code
+
+# Install Node & Yarn dependencies
+RUN yarn install && \
+    yarn global add junit-report-merger && \
+    yarn cache clean
+
+# Install trcli and Playwright
+RUN pip install --no-cache-dir trcli && \
```

Hadolint

[warning] Pin versions in pip. Instead of `pip install <package>` use `pip install <package>==<version>` or `pip install --requirement <requirements file>`

(DL3013)